enhancements/split_helm_ansible_into_their_repos.md

```diff
@@ -0,0 +1,187 @@
+
+For Helm, besides the requirement of split the repo it also has a new pkg implementation which requires to be putted in place and is in the [joelanford/helm-operator](https://github.com/joelanford/helm-operator) currently. 
+
+This proposal is decoupling both requirements, which allow a specific strategy for each one of them. However, would this option be more comfortable to move forward with both needs or the approach to address them for Helm simultaneously could be more practical?
+
+**Note** A enhancement to describe the possible options to address the Helm requirement for the new pkg implementation ought to be made.
+
+## Summary
+
+This proposal has the goal to describe how we can split Helm/Ansible projects and then, decoupled from the CLI tool. 
+
+**Note** It is a follow up of the proposal [Split SDK into multiple projects](split-sdk-into-multiple-projects.md).   
+
+## Motivation
+
+- Improve the maintainability and reusability by adopting concepts such as [separation of concerns](https://en.wikipedia.org/wiki/Separation_of_concerns) and [Single Responsibility principle](https://en.wikipedia.org/wiki/Single-responsibility_principle). 
+- Have healthier lifecycle since separately and individually, which means that we can have a single/granular releases.
+- Solve the most significant part of the problems faced with the CI. Currently, SDK repo takes to long to pass in the tests, which indeed is not just a problem to contribute to the projects but for us be able to cover more scenarios and ensure quality. 
+- Ensure that any type/language operator will be able to be built following the same base config and using a common lib which will be helpful to keep them maintained and supported as to provide more types of operators such as hybrids and done for other languages.  
+
+### Goals
+
+- The SDK CLI tool repository has just the code implementation, which is part of its domain of responsibility. 
+- A Helm Operator repository with all code implementation which is specific to it and is part of its domain of responsibility
+- An Ansible Operator repository with all code implementation which is specific to it and is part of its domain of responsibility
+- An SDK Operator Lib which contains the standard code that should be used to generate all types of Operators. (e.g. utils for releases which might be expected for all, templates and the implementation of the plugins which can be used for any language/type of operator)
+- Ansible and Helm repository have their godocs generated automatically 
+- Ansible and Helm repository have their CI, makefile targets and helpers and utilities required to contribute to them individually and test the projects scaffolded for these types. 
+- Helm/Ansible binaries should be created/released from their repos and no longer from SDK CLI. 
+
+### Non-Goals
+
+- Apply the changes for new Helm based-operator implementation in the [joelanford/helm-operator](https://github.com/joelanford/helm-operator) repository.
```

Comment:
Is there a good reason to mention this in this EP? Is @joelanford's `helm-operator` implementation discussed in the SDK's docs currently?

Reply:
I think you are right @estroz. I removed it from here and just leave in the Alternatives 👍